benchmark/es/class-bench.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+const common = require('../common.js');
+const util = require('util');
+
+const A = Symbol('a');
+const B = Symbol('b');
+
+const bench = common.createBenchmark(main, {
+  type: ['class',
+         'function',
+         'function-new',
+         'function-new-instanceof',
+         'function-new-target'],
+  millions: [10]
+});
+
+class FooBase {
```

Comment:
Might be a bit easier to read if these are `ES6BaseClass`, `ES6Subclass`, `FunctionWithNewCheck`...(not necessary to be so verbose, just not `Foo` `Bar`)